src/client.ts

```diff
@@ -510,6 +526,23 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
       });
   }
 
+  parseJwt (): void {
+    const decoded: JWTDetails = jwtDecode(this._config.jwt);
```

Comment:
Does this throw an error if the jwt is malformed? If yes, maybe we should emit on `sdkError`. Although, if we through this in the constructor, there won't be any listeners yet.

Reply:
I'll check this.